frontend/src/app/utils/srGraphQLErrorMessage.tsx

```diff
@@ -23,21 +23,59 @@ export function getHeader(message: string): string {
   return getErrorText(message, "header");
 }
 
+function addEndWhiteSpace(errorArrayLength: number, index: number) {
+  return index !== errorArrayLength - 1;
+}
+
+function createSupportMailToLink(
+  msg: string,
+  index: number,
+  addEndWhiteSpace: boolean
+) {
+  return (
+    <span key={index}>
+      {" "}
+      <a href="mailto:support@simplereport.gov">{msg}</a>
+      {addEndWhiteSpace ? " " : ""}
+    </span>
+  );
+}
+
+function createContactUsLink(
+  msg: string,
+  nextMsg: string,
+  index: number,
+  addEndWhiteSpace: boolean
+) {
+  return (
+    <span key={index}>
+      {" "}
+      <a target="_blank" href="/contact-us">
+        {msg} {nextMsg}
```

Comment:
I'm not sure this will work in production to redirect to the static site - if I'm reading this correctly, this will target the link to `simplereport.gov/app/contact-us`, and the static site doesn't have that `/app` piece.

Reply:
This is true the url will need to be absolute so it needs to include the origin `https://`